### PR TITLE
Add dockerfile

### PR DESCRIPTION
**Pull request by duythien**
Hello
This is dockerfile for development and also so for production, to use it just

```
docker-compose build
docker-compose up -d
```

Currently I use https://www.phusionpassenger.com/library/ for web server rails

### CODE REVIEW

File: config/passenger/staging.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+ "nginx_config_template": "config/passenger/staging.conf.erb",
+ "port": "4107",
+ "spawn_method": "direct"
+}
```
Comment (kenneth-kip) [inline config/passenger/staging.json]: Add new line here.

File: config/passenger/development.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+ "nginx_config_template": "config/passenger/development.conf.erb",
+ "port": "80",
+ "spawn_method": "direct"
+}
```
Comment (kenneth-kip) [inline config/passenger/development.json]: Add new line here.

File: config/passenger/daily.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+ "nginx_config_template": "config/passenger/daily.conf.erb",
+ "port": "4107",
+ "spawn_method": "direct"
+}
```
Comment (kenneth-kip) [inline config/passenger/daily.json]: Add new line here.

**Comment by tistudios** [comment]
is this good to go?

**Pull request closed without merging**